rust/noosphere-storage/src/implementation/sled.rs

```diff
@@ -54,6 +58,25 @@ impl Storage for SledStorage {
     }
 }
 
+#[async_trait]
+impl ConfigurableStorage for SledStorage {
```

Comment:
It seems like this implementation is the same for both `SledStorage` and `RocksDbStorage`. Can it just be a general default implementation?

Reply:
I don't think so, they both just coincidentally have the core of their instantiation and setup in `Self::with_config` rather than calling `<Self as ConfigurableStorage>::open_with_config` from `Self::new`